R-package/R/package.R

```diff
@@ -26,3 +26,8 @@ RGF_mod <- NULL; RGF_utils <- NULL; SCP <- NULL;
     }
   }, silent=TRUE)
 }
+
+
+.onAttach <- function(libname, pkgname) {
```

Comment:
Am I right that this message is shown unconditionally each time `library(RGF)` is called? I'm afraid that it will be annoying for users that have no troubles with loading the library but have to see this message.

Reply:
There are always users who have issues with an R package that uses Python code and users that don't. Should the users that do have troubles spent hours debugging this issue? Do you have a better solution for this one? Last time that I've submitted my R packages (that use Python code) to CRAN I also spent many hours to account for the variations of Python. I currently do not have time. Feel free to modify the code so that users of all OS's and Python versions can load the R package and get informed for a potential issue.

Reply:
In my opinion, startup messages should contain only very important information that will be useful for **the majority of users**. Otherwise, such messages become annoying very fast. Moreover, this message particularly is useful only once, after users read it for the first time it becomes useless for them.

Quite common method of delivering solutions to known issues is to write down them into a special section of a README file, generally named as "Known issues/bug" or "FAQ". What do you think about that?

Reply:
I'm ok with any solution. I just wanted to add the one that I picked for my other R packages that use Python code. If you think it's better to add a "Known Issues" section, it's fine by me. Let me know so that we can adjust this PR (or you can make the modifications as well)

Reply:
I just suggested another variant to inform users about solution for a known issue. But I believe that final decision should be made by you as a maintainer of R-package and experienced R lang user. So feel free to just say that the proposed in this PR solution is commonly used among R users and we should merge it.

Reply:
we can proceed merge this PR so that I can submit the new version to CRAN

Reply:
OK, let's merge this PR.

Reply:
Give me a few days and I'll submit the new version to CRAN. I'll notify you once the new version has been accepted

Reply:
with these kind of NOTES I struggled last time when I submitted the package to CRAN and I think I won't go through this process again. So, if you have an idea how to fix this note on Windows feel free to modify 'package.R' file

[00check.log](https://github.com/RGF-team/rgf/files/7245479/00check.log)

![NOTE](https://user-images.githubusercontent.com/3055514/135116961-2979d927-cce3-4776-99a4-3126f3c671e6.png)

Reply:
> Give me a few days and I'll submit the new version to CRAN. I'll notify you once the new version has been accepted

Thank you for working on this!

> with these kind of NOTES I struggled last time when I submitted the package to CRAN

These NOTEs are show up for the latest version of the package from the `master` branch with your recent updates (bf80f68dba2b40670ed2d92d9ae3ef89eab1eced), right?

Reply:
yes just by adding this single "packageStartupMessage". I don't have time to debug this (and I don't know if I could) mainly because I don't work on Windows (I use on a daily basis Linux). I use WIndow OS only before I submit a package to CRAN to make sure it works for it too

Reply:
I guess this might be a problem specific to your local environment as I don't see any NOTEs in the logs from our Windows CI service for the latest `master` commit.
https://github.com/RGF-team/rgf/runs/3714589235?check_suite_focus=true#step:5:1254
Also, I can't reproduce that NOTE with the [R-hub builder](https://builder.r-hub.io/advanced).

Unfortunately, I don't have enough knowledge in R to debug this particular warning and continue maintaining the whole R-package alone.
I totally understand that you're lacking time for this package and probably your interests have been shifted. And there are no problems with that at all! If you don't have resources to maintain the RGF R-package anymore, I think we can add a note to the README that this package is not maintained and stop updating and releasing it on CRAN. Please let me know what do you think.

Reply:
@StrikerRUS,

you probably misunderstood my previous message. Let me clarify,

* I invested many personal hours to develop the RGF R package before the integration to the RGF Github account, therefore for any issues related to the existing code base I can search / give potential answers to R users
* the fact that I do not want to invest time to account for errors that appear in specific Python variations of an Operating System (such as windows) does not mean that I do not want to further maintain the R package

Since we use the 'reticulate' package and since this package is very popular and used by many users it is clear that many issues will appear because each user may use a different Operating System, a different Python Version, a different Python Environment (such as Miniconda). I'm not in place to have knowledge of all these different variations and I assume the CRAN policy is also the same: they just want that an R package is uploaded with minimal Python dependencies.

My suggestion for this PR is that we either close it or wait till a user with a knowledge specific to this error message can give an answer so that we can submit to CRAN.